packages/web-app-files/src/components/SideBar/FileInfo.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="file_info oc-flex oc-flex-between">
+  <div class="file_info oc-flex oc-flex-between oc-p-m">
```

Comment:
What do you think about this (same for spaces):

```suggestion
  <div class="file_info oc-flex oc-flex-between oc-px-s oc-py-m">
```


It would align vertically with the rest:

![image](https://user-images.githubusercontent.com/50302941/217515809-d47ee456-ceb2-4eed-b5d7-4afa14e88daa.png)

Reply:
done. ty